xml/System.Diagnostics/ActivityEvent.xml

```diff
@@ -14,7 +14,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Represents a text annotation associated with a collection of tags.</summary>
```

Comment:
>Represents a text annotation associated with a collection of tags [](start = 13, length = 65)

represent an event containing a name and timestamp and optionally list of tags.

Reply:
```suggestion
    <summary>Represents aan event containing a name and a timestamp, as well as an optional list of tags.</summary>
```